src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,63 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("1,2 문자열을 , 로 split 했을 때 1과 2로 잘 분리되는지 확인하는 학습 테스트 구현")
+    public void stringTest_split(){
+        //given
+        final String given = "1,2";
+        final String[] expected = {"1","2"};
+
+        //when & then
+        assertThat(given.split(",")).containsExactly(expected);
+    }
+
+    @Test
+    @DisplayName("(1,2) 값이 주어졌을 때 String의 substring() 메소드를 활용해 ()을 제거하고 1,2 반환하도록 구현")
+    public void stringTest_substring(){
+        //given
+        final String given = "(1,2)";
+        final String expected = "1,2";
+
+        //when & then
+        assertThat(given.substring(1,given.length()-1)).isEqualTo(expected);
```

Comment:
```suggestion
        //when
        String actual = given.substring(1,given.length()-1);
        
        // then
        assertThat(actual).isEqualTo(expected);
```

테스트 하기위한 로직을 명확하게 분리해도 좋을것 같아요 👍